src/content/docs/data-apis/custom-data/custom-events/collect-custom-attributes.mdx

```diff
@@ -96,6 +96,7 @@ To enable and use custom attributes for APM, follow the procedure for your <Inli
     title="Java"
   >
     Custom attribute collection is enabled by default in Java. You can collect custom attributes using XML and the Java agent APIs. These two methods can be used in conjunction with each other.
+    Note that collecting custom attributes requires that the [New Relic Java API jar](/docs/apm/agents/java-agent/api-guides/guide-using-java-agent-api) be in the application's class path. 
```

Comment:
@jtduffy I see most usages of this as a single word. Could we change this to "classpath"?

Reply:
Done